docs/docs/cmd/cli/config/config-set.md

```diff
@@ -26,6 +26,12 @@ Configure CLI to automatically display help when executing a command failed
 m365 cli config set --key showHelpOnFailure --value true
 ```
 
+Configure the default output when executing a command. Allowed values `text|json`
```

Comment:
Let's make this an example of a specific use case rather than a spec of what's possible to align with other examples we have